src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -540,31 +540,48 @@
 												  VerticalAlignment="{TemplateBinding VerticalContentAlignment}" />
 							</Border>
 
-							<!-- Padding and VerticalAlignement="Top" are here to make sure the ScrollViewer does not move when entering a second line when multiline is enabled -->
-							<ScrollViewer x:Name="ContentElement"
-										  Grid.Column="1"
-										  HorizontalScrollBarVisibility="{TemplateBinding ScrollViewer.HorizontalScrollBarVisibility}"
-										  HorizontalScrollMode="{TemplateBinding ScrollViewer.HorizontalScrollMode}"
-										  IsDeferredScrollingEnabled="{TemplateBinding ScrollViewer.IsDeferredScrollingEnabled}"
-										  IsHorizontalRailEnabled="{TemplateBinding ScrollViewer.IsHorizontalRailEnabled}"
-										  IsTabStop="False"
-										  IsVerticalRailEnabled="{TemplateBinding ScrollViewer.IsVerticalRailEnabled}"
-										  Padding="0,10"
-										  VerticalAlignment="Top"
-										  VerticalScrollMode="{TemplateBinding ScrollViewer.VerticalScrollMode}"
-										  VerticalScrollBarVisibility="{TemplateBinding ScrollViewer.VerticalScrollBarVisibility}"
-										  ZoomMode="Disabled"
-										  AutomationProperties.AccessibilityView="Raw">
-								<ScrollViewer.RenderTransform>
-									<CompositeTransform x:Name="ContentElement_CompositeTransform" />
-								</ScrollViewer.RenderTransform>
-							</ScrollViewer>
-
-							<!-- Border in place to properly vertically center the placeholder inside when it's a one-line TextBox -->
-							<!-- but keep it in the same place and at the top when it's a multiline TextBox -->
-							<Border Grid.Column="1"
-									VerticalAlignment="Center">
-								<Border Height="38">
+							<Grid Grid.Column="1">
+								<Border MinHeight="38"
```

Comment:
i already shown how to do it with Outlined style
you need to remove this 38 to replace by 40
and move border into a separate `<Border/>`